Source/Views/ButtonStyle/RoundedCornersButtonStyle.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  RoundedCornersButtonStyle.swift
+//  GithubNotifications
+//
+//  Created by Chananchida F. on 9/8/21.
+//  Copyright © 2021 Nimblehq. All rights reserved.
+//
+
+import SwiftUI
+
+struct RoundedCornersStyle: ButtonStyle {
+ 
+    var color: Color = Color.black
+    
+    func makeBody(configuration: Self.Configuration) -> some View {
+        configuration.label
+            .padding()
+            .frame(width: 155.0, height: 30.0, alignment: .center)
```

Comment:
I think we shouldn't have a fixed size inside this style.
btw, we're having the same idea, take a look at #51

Reply:
I updated the code. Please help check 🙇‍♀️